### PR TITLE
Add markdown support (read only view)

### PR DESCRIPTION
**Pull request by David-Development**
Render Markdown incl. images and inline LaTeX Code.

### CODE REVIEW

File: app/src/main/assets/markdown-viewer/markdown_template.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            text = text.replace(/&lt;/g, "<");
+            text = text.replace(/&gt;/g, ">");
+            text = text.trim();
+            //console.log(text);
+            var markedText = marked(text);
+
+            //log(markedText);
+
+            // TypeScript Support
+            const regexTypeScript1 = /<code class="lang-ts/g;
```
Comment (kaeptmblaubaer1000) [inline app/src/main/assets/markdown-viewer/markdown_template.html]: You know that `const` isn't compatible with older browsers?

File: app/src/main/java/de/luhmer/git/MarkdownViewerActivity.java [hunk 2]
```diff
@@ -0,0 +1,210 @@
+package de.luhmer.git;
```
Comment (kaeptmblaubaer1000) [inline app/src/main/java/de/luhmer/git/MarkdownViewerActivity.java]: You should probably use the `com.manichord.mgit.activities` package

File: app/src/main/assets/markdown-viewer/markdown_template.html (hunk 1 above)
Comment (David-Development) [reply app/src/main/assets/markdown-viewer/markdown_template.html]: The [Mozilla Developer Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const) claims, that `const` is supported by Android. The Markdown View uses the default Android WebView which will be updated by the PlayStore. So this shouldn't be a problem.

Comment (kaeptmblaubaer1000) [reply app/src/main/assets/markdown-viewer/markdown_template.html]: Actually, MGit supports Android down to 4.1, which isn't supported by the WebView update.
And there are people who don't update the app/uninstall it.

**Comment by kaeptmblaubaer1000** [comment]
Great! But please reformat the source code of `app/src/main/java/com/manichord/mgit/activities/MarkdownViewerActivity.java` and optimize imports of `app/src/main/java/me/sheimi/sgit/fragments/FilesFragment.java` so that they comply with the code style settings.

**Comment by David-Development** [comment]
I'm not sure what code styles you're referring to. Let me know if there's anything else you want me to change. Thanks for merging this back into the app! :)

**Comment by kaeptmblaubaer1000** [comment]
@maks please look into this

**Comment by maks** [comment]
Thank you for contributing this. 
However I have concerns around adding specific support for rendering markdown. The existing file viewing is really supposed to be for basic viewing only and its my intention to remove builtin editor from MGit going forward. I do realise that Github for example treats readme files in markup formats such as markdown specially but I'm not convinced this is the direction MGit should go.
My preference is that any PRs be done to address specific issues and that discussion of if a feature will be implemented be done in the issue prior to coding, which would hopefully save implementing features that do not have a consensus on being added. I'm sorry this has not been spelled out in the project Readme til now - I'll do that asap.

**Pull request closed without merging**